rst_files/julia_by_example.rst

```diff
@@ -145,17 +143,17 @@ The other packages ``LinearAlgebra`` and ``Statistics`` are base Julia libraries
 
 The arguments to ``plot`` are the numbers ``1,2, ..., n`` for the x-axis, a vector ``ϵ`` for the y-axis, and (optional) settings
 
-The function ``randn(n)`` returns a column vector ``n`` random draws from a normal distribution mean 0 and variance 1
+The function ``randn(n)`` returns a column vector with *n* random draws from a normal distribution mean 0 and variance 1
```

Comment:
That is markdown or something similar.

Reply:
The Markdown and rST syntax actually agree on this one (italics)